src/components/Conversations/Conversations.tsx

```diff
@@ -65,7 +72,48 @@ export default function Conversations() {
                 return;
             }
 
-            const listResults: MedicalScribeJobSummary[] = listHealthScribeJobsRsp.MedicalScribeJobSummaries;
+            let listResults: MedicalSoapJobSummary[] = listHealthScribeJobsRsp.MedicalScribeJobSummaries;
+
+            const jobPromises = listResults.map(async (job) => {
+                const healthSoapJobDetail = await getHealthScribeJob({
+                    MedicalScribeJobName: job.MedicalScribeJobName || '',
+                });
+
+                job.firstName = '';
+                if (healthSoapJobDetail.MedicalScribeJob && healthSoapJobDetail.MedicalScribeJob?.Tags) {
+                    job.firstName =
+                        healthSoapJobDetail.MedicalScribeJob?.Tags?.find((t) => t.Key === 'firstName')?.Value || '';
+                }
```

Comment:
this piece of code is almost the same four times, if possible please try to use just one if statement and inside it made the assignations

Reply:
Done.